RELEASES.md

```diff
@@ -22,10 +39,44 @@ Enhancements
 
   https://github.com/rust-ndarray/ndarray/pull/862
```

Comment:
This would fit better in the "Bug fixes" section. Also, a more descriptive label would be "Fix `Zip::indexed`..." rather than "Fix `Zip`...".